README.md

```diff
@@ -16,6 +16,7 @@
 *   [Configuration](#configuration)
 *   [Upgrading](#upgrading)
 *   [FAQ](#faq)
+*   [CONTRIBUTE](#contribute)
```

Comment:
Thanks for the in depth look. Since this PR is open to fix the Table of Contents. If you want to also add the Disclaimer in that would be great

Reply:
Updated the readme to add disclaimer. Thanks for the suggestion. 🙂